readme.md

```diff
@@ -366,7 +366,7 @@ as follows:
 
 ### Why is this named alex?
 
-It’s a nice androgynous/unisex name, it was free on npm, I like it! :smile:
```

Comment:
`It’s` is a perfect contraction of `it was`, plus I dig the tone of voice. Keeping it.